{"@t":"2023-12-19T11:34:21.7632932Z","@mt":"Acquiring MainDom.","SourceContext":"Umbraco.Cms.Core.Runtime.MainDom","ProcessId":1672,"ProcessName":"iisexpress","ThreadId":1,"ApplicationId":"37611e7557a6d2addb9de42bf30835f849c38595","MachineName":"DESKTOP-LMKLOT0","Log4NetLevel":"INFO "}
{"@t":"2023-12-19T11:34:21.8117345Z","@mt":"Acquired MainDom.","SourceContext":"Umbraco.Cms.Core.Runtime.MainDom","ProcessId":1672,"ProcessName":"iisexpress","ThreadId":1,"ApplicationId":"37611e7557a6d2addb9de42bf30835f849c38595","MachineName":"DESKTOP-LMKLOT0","Log4NetLevel":"INFO "}
{"@t":"2023-12-19T11:34:22.5282971Z","@mt":"Configured database is reporting as not being available.","@l":"Warning","@x":"Microsoft.Data.SqlClient.SqlException (0x80131904): A connection was successfully established with the server, but then an error occurred during the login process. (provider: Shared Memory Provider, error: 0 - No process is on the other end of the pipe.)\r\n ---> System.ComponentModel.Win32Exception (233): No process is on the other end of the pipe.\r\n   at Microsoft.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)\r\n   at Microsoft.Data.SqlClient.TdsParserStateObject.ReadSniError(TdsParserStateObject stateObj, UInt32 error)\r\n   at Microsoft.Data.SqlClient.TdsParserStateObject.ReadSniSyncOverAsync()\r\n   at Microsoft.Data.SqlClient.TdsParserStateObject.TryReadNetworkPacket()\r\n   at Microsoft.Data.SqlClient.TdsParserStateObject.TryPrepareBuffer()\r\n   at Microsoft.Data.SqlClient.TdsParserStateObject.TryReadByte(Byte& value)\r\n   at Microsoft.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)\r\n   at Microsoft.Data.SqlClient.TdsParser.Run(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj)\r\n   at Microsoft.Data.SqlClient.SqlInternalConnectionTds.CompleteLogin(Boolean enlistOK)\r\n   at Microsoft.Data.SqlClient.SqlInternalConnectionTds.AttemptOneLogin(ServerInfo serverInfo, String newPassword, SecureString newSecurePassword, Boolean ignoreSniOpenTimeout, TimeoutTimer timeout, Boolean withFailover)\r\n   at Microsoft.Data.SqlClient.SqlInternalConnectionTds.LoginNoFailover(ServerInfo serverInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString connectionOptions, SqlCredential credential, TimeoutTimer timeout)\r\n   at Microsoft.Data.SqlClient.SqlInternalConnectionTds.OpenLoginEnlist(TimeoutTimer timeout, SqlConnectionString connectionOptions, SqlCredential credential, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance)\r\n   at Microsoft.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken, DbConnectionPool pool)\r\n   at Microsoft.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)\r\n   at Microsoft.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at Microsoft.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry, SqlConnectionOverrides overrides)\r\n   at Microsoft.Data.SqlClient.SqlConnection.Open(SqlConnectionOverrides overrides)\r\n   at Umbraco.Extensions.DbConnectionExtensions.IsAvailable(IDbConnection connection)\r\nClientConnectionId:6c928b2b-6c35-496a-8751-a0f94669831b\r\nError Number:233,State:0,Class:20","SourceContext":"object","ProcessId":1672,"ProcessName":"iisexpress","ThreadId":1,"ApplicationId":"37611e7557a6d2addb9de42bf30835f849c38595","MachineName":"DESKTOP-LMKLOT0","Log4NetLevel":"WARN "}
{"@t":"2023-12-19T11:34:23.5579719Z","@mt":"Configured database is reporting as not being available.","@l":"Warning","@x":"Microsoft.Data.SqlClient.SqlException (0x80131904): A connection was successfully established with the server, but then an error occurred during the login process. (provider: Shared Memory Provider, error: 0 - No process is on the other end of the pipe.)\r\n ---> System.ComponentModel.Win32Exception (233): No process is on the other end of the pipe.\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at Microsoft.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry, SqlConnectionOverrides overrides)\r\n   at Microsoft.Data.SqlClient.SqlConnection.Open(SqlConnectionOverrides overrides)\r\n   at Umbraco.Extensions.DbConnectionExtensions.IsAvailable(IDbConnection connection)\r\nClientConnectionId:6c928b2b-6c35-496a-8751-a0f94669831b\r\nError Number:233,State:0,Class:20","SourceContext":"object","ProcessId":1672,"ProcessName":"iisexpress","ThreadId":1,"ApplicationId":"37611e7557a6d2addb9de42bf30835f849c38595","MachineName":"DESKTOP-LMKLOT0","Log4NetLevel":"WARN "}
{"@t":"2023-12-19T11:34:24.5683768Z","@mt":"Configured database is reporting as not being available.","@l":"Warning","@x":"Microsoft.Data.SqlClient.SqlException (0x80131904): A connection was successfully established with the server, but then an error occurred during the login process. (provider: Shared Memory Provider, error: 0 - No process is on the other end of the pipe.)\r\n ---> System.ComponentModel.Win32Exception (233): No process is on the other end of the pipe.\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at Microsoft.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry, SqlConnectionOverrides overrides)\r\n   at Microsoft.Data.SqlClient.SqlConnection.Open(SqlConnectionOverrides overrides)\r\n   at Umbraco.Extensions.DbConnectionExtensions.IsAvailable(IDbConnection connection)\r\nClientConnectionId:6c928b2b-6c35-496a-8751-a0f94669831b\r\nError Number:233,State:0,Class:20","SourceContext":"object","ProcessId":1672,"ProcessName":"iisexpress","ThreadId":1,"ApplicationId":"37611e7557a6d2addb9de42bf30835f849c38595","MachineName":"DESKTOP-LMKLOT0","Log4NetLevel":"WARN "}
{"@t":"2023-12-19T11:34:25.5844969Z","@mt":"Configured database is reporting as not being available.","@l":"Warning","@x":"Microsoft.Data.SqlClient.SqlException (0x80131904): A connection was successfully established with the server, but then an error occurred during the login process. (provider: Shared Memory Provider, error: 0 - No process is on the other end of the pipe.)\r\n ---> System.ComponentModel.Win32Exception (233): No process is on the other end of the pipe.\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at Microsoft.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry, SqlConnectionOverrides overrides)\r\n   at Microsoft.Data.SqlClient.SqlConnection.Open(SqlConnectionOverrides overrides)\r\n   at Umbraco.Extensions.DbConnectionExtensions.IsAvailable(IDbConnection connection)\r\nClientConnectionId:6c928b2b-6c35-496a-8751-a0f94669831b\r\nError Number:233,State:0,Class:20","SourceContext":"object","ProcessId":1672,"ProcessName":"iisexpress","ThreadId":1,"ApplicationId":"37611e7557a6d2addb9de42bf30835f849c38595","MachineName":"DESKTOP-LMKLOT0","Log4NetLevel":"WARN "}
{"@t":"2023-12-19T11:34:26.5989877Z","@mt":"Configured database is reporting as not being available.","@l":"Warning","@x":"Microsoft.Data.SqlClient.SqlException (0x80131904): A connection was successfully established with the server, but then an error occurred during the login process. (provider: Shared Memory Provider, error: 0 - No process is on the other end of the pipe.)\r\n ---> System.ComponentModel.Win32Exception (233): No process is on the other end of the pipe.\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at Microsoft.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry, SqlConnectionOverrides overrides)\r\n   at Microsoft.Data.SqlClient.SqlConnection.Open(SqlConnectionOverrides overrides)\r\n   at Umbraco.Extensions.DbConnectionExtensions.IsAvailable(IDbConnection connection)\r\nClientConnectionId:6c928b2b-6c35-496a-8751-a0f94669831b\r\nError Number:233,State:0,Class:20","SourceContext":"object","ProcessId":1672,"ProcessName":"iisexpress","ThreadId":1,"ApplicationId":"37611e7557a6d2addb9de42bf30835f849c38595","MachineName":"DESKTOP-LMKLOT0","Log4NetLevel":"WARN "}
{"@t":"2023-12-19T11:34:26.6494460Z","@mt":"Failed to detected SqlServer version.","@l":"Error","@x":"Microsoft.Data.SqlClient.SqlException (0x80131904): A connection was successfully established with the server, but then an error occurred during the login process. (provider: Shared Memory Provider, error: 0 - No process is on the other end of the pipe.)\r\n ---> System.ComponentModel.Win32Exception (233): No process is on the other end of the pipe.\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)\r\n   at Microsoft.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at Microsoft.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry, SqlConnectionOverrides overrides)\r\n   at Microsoft.Data.SqlClient.SqlConnection.Open(SqlConnectionOverrides overrides)\r\n   at Umbraco.Cms.Persistence.SqlServer.Services.SqlServerSyntaxProvider.GetSetVersion(String connectionString, String providerName, ILogger logger)\r\nClientConnectionId:6c928b2b-6c35-496a-8751-a0f94669831b\r\nError Number:233,State:0,Class:20","SourceContext":"Umbraco.Cms.Persistence.SqlServer.Services.SqlServerSyntaxProvider","ProcessId":1672,"ProcessName":"iisexpress","ThreadId":1,"ApplicationId":"37611e7557a6d2addb9de42bf30835f849c38595","MachineName":"DESKTOP-LMKLOT0","Log4NetLevel":"ERROR"}
{"@t":"2023-12-19T11:34:26.7186241Z","@mt":"Boot Failed","@l":"Error","@x":"Umbraco.Cms.Core.Exceptions.BootFailedException: A connection string is configured but Umbraco could not connect to the database.\r\n   at Umbraco.Cms.Infrastructure.Runtime.RuntimeState.DetermineRuntimeLevel()\r\n   at Umbraco.Cms.Infrastructure.Runtime.CoreRuntime.DetermineRuntimeLevel()","SourceContext":"Umbraco.Cms.Infrastructure.Runtime.CoreRuntime","ProcessId":1672,"ProcessName":"iisexpress","ThreadId":1,"ApplicationId":"37611e7557a6d2addb9de42bf30835f849c38595","MachineName":"DESKTOP-LMKLOT0","Log4NetLevel":"ERROR"}
{"@t":"2023-12-19T11:42:06.0339262Z","@mt":"Acquiring MainDom.","SourceContext":"Umbraco.Cms.Core.Runtime.MainDom","ProcessId":4272,"ProcessName":"iisexpress","ThreadId":1,"ApplicationId":"37611e7557a6d2addb9de42bf30835f849c38595","MachineName":"DESKTOP-LMKLOT0","Log4NetLevel":"INFO "}
{"@t":"2023-12-19T11:42:06.0903878Z","@mt":"Acquired MainDom.","SourceContext":"Umbraco.Cms.Core.Runtime.MainDom","ProcessId":4272,"ProcessName":"iisexpress","ThreadId":1,"ApplicationId":"37611e7557a6d2addb9de42bf30835f849c38595","MachineName":"DESKTOP-LMKLOT0","Log4NetLevel":"INFO "}
{"@t":"2023-12-19T11:42:08.7352965Z","@mt":"No last synced Id found, this generally means this is a new server/install. A cold boot will be triggered.","@l":"Warning","SourceContext":"Umbraco.Cms.Infrastructure.Sync.SyncBootStateAccessor","ProcessId":4272,"ProcessName":"iisexpress","ThreadId":1,"ApplicationId":"37611e7557a6d2addb9de42bf30835f849c38595","MachineName":"DESKTOP-LMKLOT0","Log4NetLevel":"WARN "}
{"@t":"2023-12-19T11:42:10.5524527Z","@mt":"uSync: Startup Complete {elapsed}ms","elapsed":13,"SourceContext":"uSync.BackOffice.Notifications.uSyncApplicationStartingHandler","ProcessId":4272,"ProcessName":"iisexpress","ThreadId":1,"ApplicationId":"37611e7557a6d2addb9de42bf30835f849c38595","MachineName":"DESKTOP-LMKLOT0","Log4NetLevel":"INFO "}
{"@t":"2023-12-19T11:42:10.5537928Z","@mt":"Application started. Press Ctrl+C to shut down.","SourceContext":"Microsoft.Hosting.Lifetime","ProcessId":4272,"ProcessName":"iisexpress","ThreadId":1,"ApplicationId":"37611e7557a6d2addb9de42bf30835f849c38595","MachineName":"DESKTOP-LMKLOT0","Log4NetLevel":"INFO "}
{"@t":"2023-12-19T11:42:10.5539307Z","@mt":"Hosting environment: {envName}","envName":"Development","SourceContext":"Microsoft.Hosting.Lifetime","ProcessId":4272,"ProcessName":"iisexpress","ThreadId":1,"ApplicationId":"37611e7557a6d2addb9de42bf30835f849c38595","MachineName":"DESKTOP-LMKLOT0","Log4NetLevel":"INFO "}
{"@t":"2023-12-19T11:42:10.5539708Z","@mt":"Content root path: {contentRoot}","contentRoot":"C:\\Controlled Folder\\Umbraco\\Digital Agency\\DigitalAgency.Web","SourceContext":"Microsoft.Hosting.Lifetime","ProcessId":4272,"ProcessName":"iisexpress","ThreadId":1,"ApplicationId":"37611e7557a6d2addb9de42bf30835f849c38595","MachineName":"DESKTOP-LMKLOT0","Log4NetLevel":"INFO "}
{"@t":"2023-12-19T11:42:12.0513560Z","@mt":"Registered with MainDom, localContentDbExists? {LocalContentDbExists}, localMediaDbExists? {LocalMediaDbExists}","LocalContentDbExists":true,"LocalMediaDbExists":true,"SourceContext":"Umbraco.Cms.Infrastructure.PublishedCache.PublishedSnapshotService","RequestId":"40000058-0005-dc00-b63f-84710c7967bb","RequestPath":"/","ProcessId":4272,"ProcessName":"iisexpress","ThreadId":7,"ApplicationId":"37611e7557a6d2addb9de42bf30835f849c38595","MachineName":"DESKTOP-LMKLOT0","Log4NetLevel":"INFO ","HttpRequestId":"924045e9-5565-4cba-aa60-ae885fc98ef4","HttpRequestNumber":1,"HttpSessionId":"0"}
{"@t":"2023-12-19T11:42:12.0578343Z","@mt":"Creating the content store, localContentDbExists? {LocalContentDbExists}","LocalContentDbExists":true,"SourceContext":"Umbraco.Cms.Infrastructure.PublishedCache.PublishedSnapshotService","RequestId":"40000058-0005-dc00-b63f-84710c7967bb","RequestPath":"/","ProcessId":4272,"ProcessName":"iisexpress","ThreadId":7,"ApplicationId":"37611e7557a6d2addb9de42bf30835f849c38595","MachineName":"DESKTOP-LMKLOT0","Log4NetLevel":"INFO ","HttpRequestId":"924045e9-5565-4cba-aa60-ae885fc98ef4","HttpRequestNumber":1,"HttpSessionId":"0"}
{"@t":"2023-12-19T11:42:12.0632902Z","@mt":"Creating the media store, localMediaDbExists? {LocalMediaDbExists}","LocalMediaDbExists":true,"SourceContext":"Umbraco.Cms.Infrastructure.PublishedCache.PublishedSnapshotService","RequestId":"40000058-0005-dc00-b63f-84710c7967bb","RequestPath":"/","ProcessId":4272,"ProcessName":"iisexpress","ThreadId":7,"ApplicationId":"37611e7557a6d2addb9de42bf30835f849c38595","MachineName":"DESKTOP-LMKLOT0","Log4NetLevel":"INFO ","HttpRequestId":"924045e9-5565-4cba-aa60-ae885fc98ef4","HttpRequestNumber":1,"HttpSessionId":"0"}
{"@t":"2023-12-19T11:42:13.4593230Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Loading content from database","TimingId":"a68acd9","SourceContext":"Umbraco.Cms.Core.Logging.ProfilingLogger","RequestId":"40000058-0005-dc00-b63f-84710c7967bb","RequestPath":"/","ProcessId":4272,"ProcessName":"iisexpress","ThreadId":7,"ApplicationId":"37611e7557a6d2addb9de42bf30835f849c38595","MachineName":"DESKTOP-LMKLOT0","Log4NetLevel":"INFO ","HttpRequestId":"924045e9-5565-4cba-aa60-ae885fc98ef4","HttpRequestNumber":1,"HttpSessionId":"0"}
{"@t":"2023-12-19T11:42:15.3784097Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Completed.","Duration":1920,"TimingId":"a68acd9","SourceContext":"Umbraco.Cms.Core.Logging.ProfilingLogger","RequestId":"40000058-0005-dc00-b63f-84710c7967bb","RequestPath":"/","ProcessId":4272,"ProcessName":"iisexpress","ThreadId":7,"ApplicationId":"37611e7557a6d2addb9de42bf30835f849c38595","MachineName":"DESKTOP-LMKLOT0","Log4NetLevel":"INFO ","HttpRequestId":"924045e9-5565-4cba-aa60-ae885fc98ef4","HttpRequestNumber":1,"HttpSessionId":"0"}
{"@t":"2023-12-19T11:42:15.4660834Z","@mt":"{StartMessage} [Timing {TimingId}]","StartMessage":"Loading media from database","TimingId":"db048fe","SourceContext":"Umbraco.Cms.Core.Logging.ProfilingLogger","RequestId":"40000058-0005-dc00-b63f-84710c7967bb","RequestPath":"/","ProcessId":4272,"ProcessName":"iisexpress","ThreadId":7,"ApplicationId":"37611e7557a6d2addb9de42bf30835f849c38595","MachineName":"DESKTOP-LMKLOT0","Log4NetLevel":"INFO ","HttpRequestId":"924045e9-5565-4cba-aa60-ae885fc98ef4","HttpRequestNumber":1,"HttpSessionId":"0"}
{"@t":"2023-12-19T11:42:15.5606438Z","@mt":"{EndMessage} ({Duration}ms) [Timing {TimingId}]","EndMessage":"Completed.","Duration":94,"TimingId":"db048fe","SourceContext":"Umbraco.Cms.Core.Logging.ProfilingLogger","RequestId":"40000058-0005-dc00-b63f-84710c7967bb","RequestPath":"/","ProcessId":4272,"ProcessName":"iisexpress","ThreadId":7,"ApplicationId":"37611e7557a6d2addb9de42bf30835f849c38595","MachineName":"DESKTOP-LMKLOT0","Log4NetLevel":"INFO ","HttpRequestId":"924045e9-5565-4cba-aa60-ae885fc98ef4","HttpRequestNumber":1,"HttpSessionId":"0"}
